package.json

```diff
@@ -11,29 +11,39 @@
     "node": ">= 8.3.0"
   },
   "scripts": {
-    "build": "babel src -d lib --ignore __tests__ && BABEL_ENV=esm babel src --out-dir es --ignore __tests__",
+    "build":
+      "babel src -d lib --ignore __tests__ && BABEL_ENV=esm babel src --out-dir es --ignore __tests__",
     "check": "flow check",
     "tdd": "jest --watch",
-    "test": "npm run lint && jest",
-    "lint": "npm run check && eslint . && prettier --list-different --ignore-path .eslintignore '**/*.{json,css,md}'",
-    "format": "eslint . --fix && prettier --write --ignore-path .eslintignore '**/*.{json,css,md}'",
-    "precommit": "lint-staged"
+    "test": "npm run lint && jest --coverage",
+    "lint":
+      "npm run check && eslint . && prettier --list-different --ignore-path .eslintignore '**/*.{json,css,md}'",
+    "format":
+      "eslint . --fix && prettier --write --ignore-path .eslintignore '**/*.{json,css,md}'",
+    "precommit": "lint-staged",
+    "prepublishOnly": "npm run build",
+    "semantic-release": "semantic-release",
```

Comment:
it's not _that_ bad to just do `node_modules/.bin/whatever` :p

Reply:
the SR cli just does that. I wanna add some of it into the 4c cli scaffolding thats less noisy